src/Bicep.Core/Diagnostics/Diagnostic.cs

```diff
@@ -9,15 +10,20 @@ namespace Bicep.Core.Diagnostics
     [DebuggerDisplay("Level = {" + nameof(Level) + "}, Code = {" + nameof(Code) + "}, Message = {" + nameof(Message) + "}")]
     public class Diagnostic : IDiagnostic
     {
-        public Diagnostic(TextSpan span, DiagnosticLevel level, string code, string message, DiagnosticLabel? label = null)
+        public Diagnostic(TextSpan span, DiagnosticLevel level, string code, string message,
+                            Uri? documentationUri = null, DiagnosticLabel? label = null)
         {
             Span = span;
             Level = level;
             Code = code;
             Message = message;
             Label = label;
+            Uri = documentationUri;
```

Comment:
Should we include the URI in the CLI logging message? Otherwise someone compiling via CLI only - e.g. in CI/CD won't see this info:

https://github.com/Azure/bicep/blob/0000f3b7bb693cd206ab313c8d9ed972f1e3aeaf/src/Bicep.Cli/Logging/BicepDiagnosticLogger.cs#L28

Reply:
The customer who requested this feature didn't want the URI in the CLI. But I'd agree it should be there.

Reply:
Also agree.